pkg/doc/did/doc.go

```diff
@@ -424,3 +466,23 @@ func populateRawAuthentications(vms []VerificationMethod) []interface{} {
 
 	return rawAuthentications
 }
+
+func populateRawProofs(proofs []Proof) []interface{} {
```

Comment:
in that case, it will be better to create only one function 
*  we can avoid copying a big structure to the function stack (if arg without a pointer)
*  we can prevent moving it to the heap (if arg is a pointer)
```
func populateRawProofs(proofs []Proof) []interface{} {
	rawProofs := make([]interface{},0,len(proofs))
	for _, p := range proofs {
		rawProofs = append(rawProofs, map[string]interface{}{
			jsonldType : p.Type,
			jsonldCreated : p.Created,
			jsonldCreator : p.Creator,
			jsonldProofValue : p.ProofValue,
			jsonldDomain : p.Domain,
			jsonldNonce : p.Nonce,
		})
	}
	return rawProofs
}
```

Reply:
Done